docs/6-rag/5-rag-pipelines.md

```diff
@@ -0,0 +1,120 @@
+# 🌳 Automating RAG with KFP Pipelines
+
+Your document intelligence RAG system works brilliantly in notebooks, but what happens when you need to process hundreds of research papers for your educational platform? 
+Manual execution doesn't scale, and students need reliable, always-available intelligent document processing.
+
+## 🔍 What is Kubeflow Pipelines (KFP)?
+
+**Kubeflow Pipelines (KFP)** is a platform designed for building and deploying portable, scalable machine learning pipelines using containers. Think of it as a sophisticated workflow orchestrator that transforms your experimental RAG system into a production-grade platform that can handle complex academic documents automatically, reliably, and at scale.
+
+While notebooks were perfect for exploring document intelligence capabilities, production systems require automation, scalability, error recovery, performance monitoring, and consistent results across runs. 
```

Comment:
```suggestion
KFP has what production systems require: automation, scalability, error recovery, performance monitoring, and consistent results across runs. It provides all these production-grade features so you can focus on your AI logic rather than infrastructure management.
```